### ISSUE TITLE
install_fedora failes due to "invalid initramfs"

### ISSUE THREAD

**Issue by fneddy** [Rudd-O/zfs-fedora-installer#34]
install_fedora failes due to "invalid initramfs"

```bash
        tank                                         ONLINE
          luks-22cb9102-58b4-4f9a-877c-8ea07c6d38ce  ONLINE
12m33.45  II  Checking / creating datasets.
12m33.48  II  Checking / creating swap zvol.
12m33.49  II  Mounting virtual and physical file systems.
12m33.52  II  Filesystem context complete.
dracut: No '/dev/log' or 'logger' included for syslog logging
12m53.30  II  Rewinding stack of actions.
12m53.84  II  Rewind complete.
12m53.84  II  Rewinding stack of actions.
12m54.86  II  Rewind complete.
12m54.86  EE  Unexpected error
Traceback (most recent call last):
  File "./src/installfedoraonzfs/__init__.py", line 1135, in install_fedora
    locals()[stage]()
  File "./src/installfedoraonzfs/__init__.py", line 955, in reload_chroot
    assert 0, 'Invalid initramfs %s' % initrd
AssertionError: Invalid initramfs /var/lib/zfs-fedora-installer/tank/boot/initramfs-5.4.8-200.fc31.x86_64.img
Traceback (most recent call last):
  File "./install-fedora-on-zfs", line 10, in <module>
    sys.exit(installfedoraonzfs.install_fedora_on_zfs())
  File "./src/installfedoraonzfs/__init__.py", line 1237, in install_fedora_on_zfs
    workdir=args.workdir,
  File "./src/installfedoraonzfs/__init__.py", line 1135, in install_fedora
    locals()[stage]()
  File "./src/installfedoraonzfs/__init__.py", line 955, in reload_chroot
    assert 0, 'Invalid initramfs %s' % initrd
AssertionError: Invalid initramfs /var/lib/zfs-fedora-installer/tank/boot/initramfs-5.4.8-200.fc31.x86_64.img


```

**Comment by Rudd-O** [comment]
This has been fixed in the latest staging branch commit.

**Comment by Rudd-O** [comment]
I will close the issue once the fix is merged into master, which must wait until I have done the proper QA across Fedora distributions.

**Comment by Rudd-O** [comment]
Pushed fix to master.
